dpnp/backend/kernels/dpnp_krnl_common.cpp

```diff
@@ -852,15 +852,15 @@ DPCTLSyclEventRef dpnp_matmul_c(DPCTLSyclQueueRef q_ref,
     {
         // using std::max for these ldx variables is required by math library
         const std::int64_t lda =
-            std::max<size_t>(1UL, size_k); // First dimensions of array_1
+            std::max<size_t>(1UL, size_n); // First dimensions of array_1
         const std::int64_t ldb =
-            std::max<size_t>(1UL, size_n); // First dimensions of array_2
+            std::max<size_t>(1UL, size_k); // First dimensions of array_2
```

Comment:
Comment have to be changed. Perhaps variable could also be renamed for clarity. Instead of `lda` I would suggest `ld_array_2`, and instead of current `ldb` use `ld_array_1`.